install.py

```diff
@@ -60,7 +60,7 @@ def main():
 
     tmp_dir = tempfile.mkdtemp()
     try:
-        binfile=ramalama
```

Comment:
Luckily this one didn't make our stable branch... So it's not effecting anyone right now

Reply:
Temping to take this one in a separate PR to fix main...